src/app/frontend/zerostate/zerostate.scss

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+$primary-color: #326DE6;
+
+.kd-zerostate-content {
+  position: fixed;
+  height: 90%;
+  width: 100%;
+}
+
+.kd-zerostate-card-header {
+  background: $primary-color;
+  min-height: 100px;
+}
+
+.kd-zerostate-card-logo {
+  height: 80px;
+  width: 80px;
+
+  padding: 0 0 0 15px;
+}
+
+.kd-zerostate-deploy-card {
+  min-width: 300px;
+  color: #FFF;
+}
+
+.kd-zerostate-deploy-bt {
+  font-size: 12px;
+}
+
+.kd-zerostate-lm-list {
+  margin-top: 10px;
+
+  > .kd-zerostate-lm-list-item {
+    padding: 0 !important;
```

Comment:
It is not needed. It can be extracted to separate rule. Only difference is that now this rule has more narrowed scope. After extraction scope will be bigger.